Docs/developer_guide/style_guide.md

```diff
@@ -33,6 +33,7 @@ Line length: Preferably keep lines shorter than 80 characters. Always keep lines
     - Standard library imports
     - Related third party imports
     - Local apps/library specific imports
+      - Slicer application imports and local/module imports may be grouped independently.

```

Comment:
Agreed. The re-ordering of import has been done to be consistent with the style but the continuous integration does not enforce or validate anything yet.

This will done as part of https://github.com/Slicer/Slicer/issues/6272